app/models/statement.rb

```diff
@@ -78,4 +83,8 @@ def rows_for_order_detail(order_detail)
     statement_rows.where(order_detail_id: order_detail.id)
   end
 
+  def to_s
```

Comment:
I'm going back and forth on how we should do this. I don't love making this the `to_s`, which could be used in a broader context and this format wouldn't necessarily make sense.

We could do what we do for `AccountUser` where it's in a case statement. That might be fine to continue for now.

Another idea would be something like `to_log_s` and we could do a `respond_to?(:to_log_s)` check. 

Yet another idea would be to create `LogEvents::StatementPresenter` and then a `LogEvents.const_get("#{object.class}Presenter")` with a rescue which falls back.

I'm just brainstorming right now. What are your thoughts?

Reply:
I like the `to_log_s` idea, if we implement it we should change this method tho
```
def loggable_to_s
    case loggable
    when AccountUser
      "#{loggable.account} / #{loggable.user}"
    else
      loggable.to_s
    end
  end
```
Change the `loggable.to_s` to `loggable.to_log_s` and refactor the ones we are logging right now.
Would this be ok?